app/assets/stylesheets/views/_homepage.scss

```diff
@@ -257,20 +237,32 @@
   }
 }
 
-// Created a custom section and header pattern to
-// - display a border-top of 2px (to replace .home-divider)
-// - margin-bottom of 30px (both mobile and desktop)
-// - padding-top to create a gap between border and heading
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+}
 
 .homepage-section {
-  margin: govuk-spacing(4) 0 govuk-spacing(8);
+  padding: govuk-spacing(4) 0 govuk-spacing(6);
+
+  &.homepage-section.homepage-section--links-and-search {
+    padding: govuk-spacing(6) 0 govuk-spacing(8);
+  }
+
+  &.homepage-section.homepage-section--services-and-info {
+    padding: govuk-spacing(6) 0 govuk-spacing(6);
+  }
 }
 
 .homepage-section__heading {
   border-top-style: solid;
   border-top-width: 2px;
   margin: 0 0 govuk-spacing(6);
   padding: govuk-spacing(4) 0 0;
+
+  &.homepage-section__heading--border-none {
```

Comment:
DOes this need to be attached to the above class? Could it be separate below this class declaration?

Reply:
You could write it separately but my take was that it's a modifier for `homepage-section__heading` so might make it more readable to nest it alongside the base class for it.

Reply:
I'm gonna be a bit bullish here as it sounds like this is a code style thing, sorry 😬 

On the GDS Way we suggest formatting our CSS in such a way that modifiers and elements off of a block are separate declarations so that they're more findable both in terms of codebase-wide searching and skimming through files. I can try to find the entry about it. I'd suggest changing this to be a separate CSS declaration to minimise how much nesting we need to do.

If you really really don't want to do that I'll back off.

Reply:
No worries. I think it's better to be with consistent with how we write things elsewhere so have separated it out.

Reply:
I've also separate out the `homepage-section` modifiers too to keep everything the same -
```
.homepage-section {
  padding: govuk-spacing(4) 0 govuk-spacing(6);
}

.homepage-section--links-and-search {
  background: govuk-colour("light-grey");
  padding: govuk-spacing(6) 0 govuk-spacing(8);
}

.homepage-section--services-and-info {
  padding: govuk-spacing(6) 0 govuk-spacing(6);
}
```

Reply:
I appreciate your understanding in this case. If you feel like this should be different I encourage you to raise it a govuk frontenders catchup to discuss it with other devs. You are allowed to disagree, although that discussion is tricky to have in individual PRs.